app/src/main/res/values/dimens.xml

```diff
@@ -75,7 +74,7 @@ http://developer.android.com/guide/topics/appwidgets/index.html#CreatingLayout
 
     <dimen name="app_widget_card_image_size">56dp</dimen>
     <dimen name="app_widget_card_min_width">250dp</dimen>
-    <dimen name="app_widget_card_min_height">64dp</dimen>
+    <dimen name="app_widget_card_min_height">40dp</dimen>
```

Comment:
I guess I don't really see a difference with this change. Does anything about this change scream out to you, @tkashkin ?